Resources/Textures/_NF/Structures/Wallmounts/paintings.rsi/meta.json

```diff
@@ -1,12 +1,60 @@
 {
     "version": 1,
-    "license": "CC-BY-SA-3.0",
-    "copyright": "Created by erhardsteinhauer, gentlebutter & ghostprince",
+    "license": "CC-BY-SA-4.0",
```

Comment:
Some of these files had their license changed to 4.0 but then in Resources/Textures/_NF/Structures/Wallmounts/wall_hanging.rsi/meta.json  you added a new file with license 3.0

grrrr make up your mind :-)